api/v1alpha1/multiclusterservice_types.go

```diff
@@ -52,6 +54,18 @@ const (
 	ServicesReferencesValidationCondition = "ServicesReferencesValidation"
 )
 
+// Reasons are provided as utility, and not part of the declarative API.
```

Comment:
Inspired from:
https://github.com/fluxcd/source-controller/blob/034bc4962d977b27b420686e322249ead9485425/api/v1/condition_types.go#L65

Reply:
Resolving this comment to proceed with merging.